src/index.ts

```diff
@@ -94,6 +95,8 @@ export function stakingTransaction(
 
   // Create a partially signed transaction
   const psbt = new Psbt({ network });
+  psbt.setVersion(PSBT_VERSION);
```

Comment:
Not sure we need this as the inputs do not have `OP_CHECKSEQUENCEVERIFY` cc @KonradStaniec 

We need this for the withdrawal as we're spending the staking/unbonding output through the timelock path.

Reply:
so: 
- so using version `2` is required only for spending scripts with `OP_CHECKSEQUENCEVERIFY` 
- but there is not adverse effects of using version `2`. So if you use tx version `2` to spend any other script, there is no drawback of doing do.

So imo fixing all tx's to be version `2` is good approach (it simplifies stuff)

Reply:
one thing to note here, does `psbt.setVersion` sets version of the transaction in psbt or the psbt itself as those are different things. 

We should set the version of the transaction to `2`.

Reply:
so if this is about transaction, then imo we can have  `2` everywhere . minor nit: I would rename  `PSBT_VERSION`, into `TRANSACTION_VERSION` or something like that.